### ISSUE TITLE
Repeal gagging laws

### ISSUE THREAD

**Issue by Floppy** [openpolitics/manifesto#340]
From the old Something New policy page. Need turning into a manifesto proposal: 

> We will repeal the elements of the Transparency of Lobbying, Non-party Campaigning and Trade Union Administration Act 2014 that relate to third party campaigning, releasing all non-party campaigners from any spending limits where they are not calling for voters to support or oppose a particular party.

**Comment by EdCaplen** [comment]
I agree with the repealing of the gagging law, it subverts freedom of speech and ultimately creates distrust between elected and elector.
